app/src/main/res/layout/dialog_sell_information.xml

```diff
@@ -0,0 +1,195 @@
+
+    <TextView
+        android:id="@+id/textViewSellInformSecondTitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="12dp"
+        android:text="@string/sell_inform_dialog_second_title"
+        android:textAppearance="@style/UiKit.TextAppearance.Regular.Text1"
+        app:layout_constraintBottom_toTopOf="@+id/textViewSellInformSecondSubtitle"
+        app:layout_constraintEnd_toEndOf="@+id/viewSellInformTextBackground"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@+id/textViewSellInformSecondStep"
+        app:layout_constraintTop_toBottomOf="@+id/textViewSellInformFirstSubtitle" />
+
+    <TextView
+        android:id="@+id/textViewSellInformSecondSubtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="12dp"
+        android:layout_marginBottom="28dp"
+        android:text="@string/sell_inform_dialog_second_subtitle"
+        android:textAppearance="@style/UiKit.TextAppearance.Regular.Text4"
+        android:textColor="@color/mountain"
+        app:layout_constraintBottom_toTopOf="@+id/checkBoxSellInformNotShow"
+        app:layout_constraintEnd_toEndOf="@+id/viewSellInformTextBackground"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="@+id/textViewSellInformSecondTitle"
+        app:layout_constraintTop_toBottomOf="@+id/textViewSellInformSecondTitle" />
+
+    <com.google.android.material.checkbox.MaterialCheckBox
+        android:id="@+id/checkBoxSellInformNotShow"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="32dp"
+        android:layout_marginBottom="20dp"
+        android:minHeight="48dp"
+        android:text="@string/sell_inform_dialog_do_not_show"
+        android:textAppearance="@style/UiKit.TextAppearance.Regular.Text3"
+        android:textColor="@color/text_night"
+        app:buttonTint="@color/black"
+        app:layout_constraintBottom_toTopOf="@+id/buttonSellInformConfirm"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <org.p2p.uikit.components.UiKitButton
+        android:id="@+id/buttonSellInformConfirm"
+        style="@style/UiKit.Components.Button.Large"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="16dp"
+        android:layout_marginBottom="16dp"
+        android:text="@string/sell_inform_dialog_confirm_btn"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
All ids start from `buttonSellInform`. 
I think `inform` is redundant here. You don't have a conflicts with other fragments since you are using binding. 
Try to name the ids as shorter as possible but don't cut their names

Reply:
renamed, but think screen prefix better then view type prefix, easy to search fragment class cmd+shift+F. theme for weekly meeting